locales/en/messages.json

```diff
@@ -3897,16 +3897,16 @@
         "message": "Gyro Notch Filter 2"
     },
     "pidTuningGyroNotch1Frequency": {
-        "message": "Gyro Notch Filter 1 Center Frequency [Hz]"
+        "message": "Center Frequency [Hz]"
     },
     "pidTuningGyroNotch2Frequency": {
-        "message": "Gyro Notch Filter 2 Center Frequency [Hz]"
+        "message": "Center Frequency [Hz]"
     },
     "pidTuningGyroNotch1Cutoff": {
-        "message": "Gyro Notch Filter 1 Cutoff Frequency [Hz]"
+        "message": "Cutoff Frequency [Hz]"
```

Comment:
We are repeating same Now we are continuously repeating the same words and phrases in different functionalities, this is not necessary and will only give more work to the translators, I think we should unify them.

Reply:
Some like "pidTuningCutoffFrequency": {

Reply:
Yeah I agree.  But then I have to change the html also, and was concerned about that in 4.2.  
Perhaps we can accept this PR, and if time permits during RC phase make a PR that looks for similar duplications within the messages?  I think there are quite a few. Also there are phrases that are repeated where we could refer to other messages, to reduce translation time.
To he honest we are a bit short of time now, so I would be grateful if we could do a better review of the messaging once we have 4.3 out?